packages/fast-layouts-react/src/column/column.tsx

```diff
@@ -91,11 +92,9 @@ export class Column extends Foundation<ColumnHandledProps, ColumnUnhandledProps,
      * or the nearest preceding break-point if no entry exists for the current break-point
      */
     private getValueByBreakpoint<T>(breakpointSet: T[]): T {
-        if (!canUseDOM()) {
-            return breakpointSet[0];
-        }
-
-        const breakpoint: Breakpoint = identifyBreakpoint(window.innerWidth);
+        const breakpoint: Breakpoint = canUseDOM()
+            ? identifyBreakpoint(window.innerWidth)
+            : this.props.defaultBreakpoint;
```

Comment:
thanks for updating @Reza-Rajabi! Could you write a test for this scenario? If it helps, you can mimic @JamesBurnside's recent change to the Breakpoint Tracker in this package and mock out the `canUseDOM` call to be false to ensure that the column gets the expected defaultBreakpoint.

Reply:
Sure